COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -3930,15 +3968,14 @@ This lab requires two COBOL programs, CBL0008 and CBL0009 and two respective JCL
 
 *Figure  8.  Successful compile*
 
- 
 **Lab Hints**
 
 ![](Images/image169.png)
```

Comment:
The lab hints here reveals the fix for the error directly. Maybe we change it back to show `05 TLIMIT` as the previous version?

Reply:
![image169](https://user-images.githubusercontent.com/53142237/125836141-2c73a4ef-a412-494f-bb41-97fb5ec686ae.png)

Like this?

Reply:
Yup! I believe the original image169 is like that :)